src/api/app/assets/stylesheets/webui2/tokenfield.scss

```diff
@@ -0,0 +1,124 @@
+// Adapted from https://github.com/underovsky/jquery-tagsinput-revisited
```

Comment:
This one as well.

Reply:
we really need a 'how to correctly use open source' training :(